src/ui-kit/RedesignButtonSecondary.jsx

```diff
@@ -6,9 +6,12 @@ const useStyles = makeStyles(theme => ({
         height: "48px",
         borderRadius: "50px",
         padding: "16px",
+        backgroundColor: theme.palette.primary.light,
+        // primary.contrastText only has 2.07:1 contrast ratio with primary.light
```

Comment:
The color looks good to me, but maybe share the AWS amplify link with the design team to get their final sign off. If they approve, I approve! We can remove this commented out text here, though :)

Reply:
Comment removed!